JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -124,7 +141,7 @@ int main(int argc,char**argv)
   TFile* ifile = new TFile(input.c_str(),"READ");
   if (!ifile->IsOpen()) { cout<<"Can't open "<<input<<endl; return 0; }
 
-  TFile* ofile = new TFile(output.c_str(),"UPDATE");
+  TFile* ofile = new TFile(output.c_str(),"RECREATE");
```

Comment:
@Calpas I'm not wild about changing this behavior by changing "UPDATE" to "RECREATE" and removing lines 158-161. Do you really need this?